validator-service/service/Cargo.toml

```diff
@@ -0,0 +1,78 @@
+[package]
+name = "validator-ledger-service"
+version = "1.0.0"
+authors = ["MobileCoin"]
+edition = "2018"
+license = "GPL-3.0"
+
+[lib]
+name = "validator_server"
+path = "src/lib.rs"
+
+[[bin]]
+name = "validator_main"
+path = "src/bin/main.rs"
+
+[dependencies]
+mc-full-service = { path = "../../full-service"}
```

Comment:
Nit: Sort by alphabetical order or separate this dependency from the rest with an empty line.